test/e2e/config/azure-dev.yaml

```diff
@@ -165,10 +163,6 @@ providers:
         targetName: "cluster-template-conformance-presubmit-artifacts.yaml"
       - sourcePath: "${PWD}/templates/test/dev/cluster-template-custom-builds.yaml"
         targetName: "cluster-template-conformance-presubmit-artifacts-windows-containerd.yaml"
```

Comment:
not related to this change but @jsturtevant do you know why we're writing the `cluster-template-custom-builds.yaml` template above twice with two different names?